src/App.tsx

```diff
@@ -1,26 +1,55 @@
+import { FC, useEffect, useState } from 'react';
+
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+
+import { getTodos } from './api';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { setTodos } from './features/todos';
+import { selectCurrentTodo } from './features/currentTodo/currentTodoSelectors';
+
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App: FC = () => {
+  const [loading, setLoading] = useState(false);
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  const dispatch = useAppDispatch();
+
+  const selectedTodo = useAppSelector(selectCurrentTodo);
+
+  useEffect(() => {
+    setLoading(true);
+
+    getTodos()
+      .then(res => {
+        dispatch(setTodos(res));
+      })
+      .finally(() => {
+        setLoading(false);
+      });
+  }, [dispatch]);
```

Comment:
Including `dispatch` in the dependency array of `useEffect` is usually unnecessary because `dispatch` is stable and does not change between renders. Consider removing it to avoid unnecessary re-renders.